apache/superset
# fix: Datasource change updated list now includes virtual tables

### SUMMARY

Before change datasource modal didn't include virtual tables, now we've updated the api and component to render all datasets

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


<img width="1062" alt="Screen Shot 2020-12-02 at 12 41 38 PM" src="https://user-images.githubusercontent.com/27827808/100928961-c6834900-349b-11eb-8455-db03b14f689c.png">

### TEST PLAN

1. Goto explore view
2. Click hamburger menu next to datasource
3. See if there virtual tables available in the list view.
4. Click one item to be, and see if the datasource explore view updates

### ADDITIONAL INFORMATION
